### PR TITLE
Add missing `DGX` import

### PR DESCRIPTION
**Pull request by jakirkham**
This example didn't run for me without this change.

### CODE REVIEW

**Comment by jakirkham** [comment]
I guess it is also defined earlier in the blog. Anyways this is a useful piece of code to drop into a script and just run. It's nice if it just runs out of the box to cutdown on setup time.

**Comment by jacobtomlinson** [comment]
Sorry this has sat here for so long @jakirkham. Is this change still relevant?

**Comment by pentschev** [comment]
The `DGX` class has been removed as of dask-cuda 0.14. If updating is important, we should replace it by `LocalCUDACluster`.

File: _posts/2019-06-09-ucx-dgx.md [hunk 1]
```diff
@@ -619,6 +619,8 @@ import cupy
 import dask.array as da
 from dask.distributed import Client, wait
 from distributed.utils import format_time, format_bytes
+from dask_cuda import DGX
```
Comment (jacobtomlinson) [inline _posts/2019-06-09-ucx-dgx.md:622]:
```suggestion
from dask_cuda import LocalCUDACluster
```

**Comment by jakirkham** [comment]
Except we are using `DGX` below.

**Comment by jakirkham** [comment]
I looked at updating this to use `LocalCUDACluster`, but that turns into rewriting the blogpost as one of the examples shows the `DGX` code. If we need to specify a note that things have changed in newer versions, that seems fine. Though I'd prefer not to make too many changes here (it's a post from last year).

**Comment by pentschev** [comment]
The `DGX` class was a specialization of `LocalCUDACluster`. All that specialization has been moved into `LocalCUDACluster`, so if you only replace `DGX` by `LocalCUDACluster`, that should suffice. I have no strong opinions on whether we should update this, so I'll leave the final decision to you.

**Comment by jakirkham** [comment]
Yeah there is just a whole section including the [`DGX` code spelled out]( https://github.com/dask/dask-blog/blob/38ab575070abde27f1589a35f3303ef92b62ce97/_posts/2019-06-09-ucx-dgx.md#dgx ). So changing the imports doesn't make much sense in context.

I think it is ok that things are out-of-date (it's an old blog). If we wanted to update them, it would be better to do a new short blogpost that references this and notes how one would do things today.

**Comment by jsignell** [comment]
Ok so it seems like this can safely be closed. @dask/maintenance

**Pull request closed without merging**